js/scripts.js

```diff
@@ -3,21 +3,41 @@ const ham = document.querySelector('.ham');
 const xIcon = document.querySelector('.xIcon');
 const menuIcon = document.querySelector('.menuIcon');
 const main = document.querySelector('.main');
-const header = document.querySelector('.bold');
+const logo = document.querySelector('.bold');
+const header = document.getElementById('logo');
+const projs = document.querySelectorAll('.proj');
+
+function projOn() {
+  projs.forEach(
+    (proj) => {
+      proj.style.filter = 'blur(4px)';
+    },
+  );
+}
+
+function projOff() {
+  projs.forEach(
+    (proj) => {
+      proj.style.filter = 'blur(0px)';
+    },
```

Comment:
- [ ] Kindly change these and other related functions to es6 arrow functions, as required in the JavaScript best practices section.

Reply:
Hey @mikenath223 I tried to do what you told me to switch all "normal" functions to ES6 arrow functions and even everything is working just fine, I started to get a LOT of ESLint 'not defined' function errors. Probably this version of ESLint doesn't detect arrow functions?